lib/configparser.py

```diff
@@ -75,7 +75,7 @@ def parse_args(self, argv=None):
             '--distro', help='distro to be installed ex: rhel_7.4le_alpa, sles_11sp3_beta', required=True)
         parser.add_argument(
             '--install-protocol' , help='Mode of Install Protocol ex: http, ftp, nfs',required=True)
-        parser.add_argument('--fs-type' ,help='RootFS type ex: xfs, ext4, btrfs', default='xfs')
+        parser.add_argument('--fs-type' ,help='RootFS type ex: xfs, ext4, btrfs',required=True)
```

Comment:
let us have this as default value as ` default="xfs"` so it will not mandate user to pass the value and default flow is working file